src/dotnet-symbol/dotnet-symbol.csproj

```diff
@@ -3,16 +3,22 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <NoWarn>;1591;1701</NoWarn>
+    <IsPublishable>true</IsPublishable>
+    <IsPackable>true</IsPackable>
     <PackAsTool>true</PackAsTool>
-    <ToolCommandName>dotnet-symbols</ToolCommandName>
+    <!-- The package version needs to be hard coded as a stable version so "dotnet install -g dotnet-symbols" works -->
+    <PackageVersion>1.0.0</PackageVersion>
```

Comment:
>PackageVersion>1.0.0</PackageVersion> [](start = 5, length = 37)

This is very odd. How are we gonna publish nightly builds of this package if all of them have the same version?

Reply:
Yes this is, but this is a special kind of "global" tool dotnet cli package which only seems to work with no "dev" or "beta" versions.  I'm not publishing this yet until some signing issues the cli team have with these global tool packages.  Hopefully I can also figure out a better way to version.